### PR TITLE
LUN-448: Rename feedback to avoid confusion

### PR DESCRIPTION
**Pull request by lunars97**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/routes/write-exercise/components/AnswerReview.tsx [hunk 1]
```diff
@@ -55,6 +55,10 @@ const Feedback = ({ documentWithResult, submission, needsToBeRepeated }: Feedbac
     message = `${getLabels().exercises.write.feedback.almostCorrect1} „${submission}“ ${
       getLabels().exercises.write.feedback.almostCorrect2
     }`
+  } else if (!submission) {
```
Comment (SteffiStuffel) [inline src/routes/write-exercise/components/AnswerReview.tsx:58]: All these changes from your other branch LUN-417 appear here, which makes it difficult to review. Please adjust the base branch of this feature branch, see here:
https://stackoverflow.com/questions/10853935/change-branch-base

**Comment by steffenkleinle** [comment]
@lunars97 thanks for your PRs :tada:  Some minor things:
* Normally the wording for git commits should be something like `LUN-417: Rename feedback` instead of `LUN-417-rename-feedback`
* Try to give commits meaningful names of what you actually did there instead of just the branch name/issue ticket
* Code changes that belong together should normally be grouped together instead of doing many commits

File: src/routes/write-exercise/components/__tests__/AnswerReview.spec.tsx [hunk 2]
```diff
@@ -18,27 +18,31 @@ describe('Feedback', () => {
...
-      <Feedback documentWithResult={documentWithResult} submission={submission} needsToBeRepeated={needsToBeRepeated} />
+      <AnswerReview
+        documentWithResult={documentWithResult}
+        submission={submission}
+        needsToBeRepeated={needsToBeRepeated}
+      />
     )
   }
 
   it('should render correct feedback', () => {
```
Comment (sarahsporck) [inline src/routes/write-exercise/components/__tests__/AnswerReview.spec.tsx:37]: please also rename the names of the tests

```suggestion
  it('should render correct answer review', () => {
```

**Review by sarahsporck — APPROVED** [review]

File: src/routes/write-exercise/components/__tests__/AnswerReview.spec.tsx (hunk 2 above)
Comment (lunars97) [reply src/routes/write-exercise/components/__tests__/AnswerReview.spec.tsx]: done

File: src/routes/write-exercise/components/AnswerReview.tsx [hunk 3]
```diff
@@ -31,16 +31,17 @@ const StyledText = styled.Text`
...
   needsToBeRepeated: boolean
 }
 
-const Feedback = ({ vocabularyItemWithResult, submission, needsToBeRepeated }: FeedbackProps): ReactElement | null => {
+const AnswerReview = ({ vocabularyItemWithResult, submission, needsToBeRepeated }: AnswerReviewProps): ReactElement | null => {
   const { result, vocabularyItem } = vocabularyItemWithResult
   const correctSolution = `„${vocabularyItem.article.value} ${vocabularyItem.word}“`
-  const wrongWithCorrectSolution = `${getLabels().exercises.write.feedback.wrongWithSolution} ${correctSolution}`
+  const wrongSolutionLabel: string = getLabels().exercises.write.feedback.wrongWithSolution
+  const wrongWithCorrectSolution = `${wrongSolutionLabel} ${correctSolution}` as string
```
Comment (steffenkleinle) [inline src/routes/write-exercise/components/AnswerReview.tsx:44]: Why did you change this? Looks like this should not be needed and therefore reverted.

**Comment by SteffiStuffel** [comment]
Something went wrong here, with you git history. A lot of changes and commits for an other branch (LUN-417) are added to this branch and PR. Please change so only the changes from LUN-448 appear in this PR.

**Comment by LeandraH** [comment]
It might be easier to close this PR, make a new branch, cherry-pick the commits from this branch that are actually necessary, then delete this branch and open a new PR with the new branch.
